autoscheduler/scraper/tests/models_tests.py

```diff
@@ -129,7 +129,53 @@ def test_instructor_performance_calculates_from_all_sections(self):
 
         # Act
         result = Grades.objects.instructor_performance(
-            dept=subject, course_num=course_num, instructor=instructor
+            dept=subject, course_num=course_num, instructor=instructor, honors=False,
+        )
+
+        # Assert
+        self.assertEqual(expected, result)
+
+    def test_instructor_performance_ignores_honors_sections_for_non_honors_query(self):

```

Comment:
Sort of a nitpick, but can you duplicate this test and make it do an `honors=True` query and assert that it got the correct honors grade?

Reply:
No, that's valid. Will add.